lib/web_ui/test/ui/vertices_test.dart

```diff
@@ -17,6 +18,12 @@ void main() {
 }
 
 void testMain() {
+  bool assertsEnabled = false;
```

Comment:
I am not super familiar with how tests run in the engine, but in the framework they always run with asserts enabled. Is there evidence that that's not true here? Otherwise, we should simplify this remove this logic. If not, is it at least guaranteed that these tests in some environments run with asserts enabled so these tests actually test what we want them to test?

Reply:
In most cases asserts are enabled and this logic was not required, but there were at least one set of tests that were failing (without this logic) because asserts were not enabled (and it appeared to be compiling in release mode?) 
  I copied this assert enabled testing logic from other testing of asserts within the engine.